lib/shoryuken/client.rb

```diff
@@ -9,15 +9,15 @@ def queues(name)
       end
 
       def sns
-        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns_endpoint))
+        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns))
       end
 
       def sns_arn
         @sns_arn ||= SnsArn
       end
 
       def sqs
-        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs_endpoint))
+        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs))
```

Comment:
@phstc the below changes to `aws_client_options` preserve the old `sqs_endpoint` behaviour as well as introducing my new behaviour, there is no breaking change.

Unless `aws_client_options` is accessed outside this class? It shouldn't be, as it's private.